collect_app/src/test/java/org/odk/collect/android/widgets/support/FakeWaitingForDataRegistry.java

```diff
@@ -0,0 +1,27 @@
+package org.odk.collect.android.widgets.support;
+
+import org.javarosa.core.model.FormIndex;
+import org.odk.collect.android.widgets.utilities.WaitingForDataRegistry;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FakeWaitingForDataRegistry implements WaitingForDataRegistry {
+
+    public List<FormIndex> waiting = new ArrayList<>();
```

Comment:
This doesn't much matter for tests but note that there's only ever one widget waiting for data.